webrtc.html

```diff
@@ -3284,39 +3284,59 @@ <h2>Attributes</h2>
             <dl data-link-for="RTCOfferOptions" data-dfn-for="RTCOfferOptions" class="dictionary-members">
               <dt><dfn>offerToReceiveAudio</dfn> of type <span class="idlMemberType"><a>boolean</a></span></dt>
               <dd>
-                <p>
-                  Whenever this is given a non-false value, and the
-                  <code><a>RTCPeerConnection</a></code> has no non-stopped
-                  "sendrecv" or "recvonly" audio transceivers, createOffer()
-                  MUST as its first step invoke the equivalent of
-                  <code>addTransceiver("audio")</code> on the
-                  <code><a>RTCPeerConnection</a></code> object, except that this
-                  MUST NOT <a>Update the negotiation-needed flag</a>,  and,
-                  provided this does not fail, proceed with createOffer()'s
-                  regular steps.
-                </p>
-                <p>
-                  In all other situations, it will be disregarded.
-                </p>
+                <p>Offer to receive audio even though no audio will be sent.
```

Comment:
actually it does not. When you add a track and set offerToReceiveAudio: true audio will be sent. While this is helpful as a  rationale I'd suggest removing it.

Reply:
I just wanted a short descriptive line about what it does before we start defining the behavior. But I agree that it must be correct. Do you have a suggestion for a sentence?

Reply:
i've removed it in #1683 after not being happy with all the "controls the receive behaviour" variants I came up with :-/

Reply:
what about a short description at the beginning of the section? Also a good place to say "use transceivers instead" ;-)

Reply:
That probably works. I was thinking about some common section to make it easier to deal with the problem you mention here https://github.com/w3c/webrtc-pc/pull/1672/files#r155310994